packages/apps/plugins/plugin-sketch/project.json

```diff
@@ -7,7 +7,13 @@
   "sourceRoot": "packages/apps/plugins/plugin-sketch/src",
   "projectType": "library",
   "targets": {
-    "build": {},
+    "build": {
+      "dependsOn": [
```

Comment:
I think this overwrites the defaults so it should include compile and `^build` as well